source/_integrations/manual.markdown

```diff
@@ -30,13 +30,15 @@ code:
   description: >
     If defined, specifies a code to enable or disable the alarm in the frontend.
     Only one of **code** and **code_template** can be specified.
+    Using secrets to store code is advised. (https://www.home-assistant.io/docs/configuration/secrets/)
```

Comment:
This applies almost any variable/configuration option we provide across our documentation.
Including: All passwords, all tokens, all pins, all mac address, and a lot more.

While I appreciate your goal here, we cannot add this line to every single configuration option. It will become a real mess is we start doing that.